packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -1200,48 +1210,68 @@ describe.each([
             ...table,
             schema: {
               ...table.schema,
-              aux: {
-                name: "aux",
-                relationshipType: RelationshipType.ONE_TO_MANY,
-                type: FieldType.LINK,
-                tableId: auxTable._id!,
-                fieldName: "fk_aux",
-                constraints: { type: "array" },
-              },
+              ...links.reduce<TableSchema>((acc, c) => {
+                acc[c.name] = {
+                  name: c.name,
+                  relationshipType: RelationshipType.ONE_TO_MANY,
+                  type: FieldType.LINK,
+                  tableId: c.tableId,
+                  fieldName: c.fk,
+                  constraints: { type: "array" },
+                }
+                return acc
+              }, {}),
             },
           })
-          // Refetch auxTable
-          auxTable = await config.api.table.get(auxTable._id!)
+          return table
+        }
 
-          const view = await config.api.viewV2.create({
-            name: "view a",
-            tableId: table._id!,
-            schema: {
-              aux: {
-                visible: true,
-                columns: {
-                  name: { visible: true, readonly: true },
-                  age: { visible: true, readonly: true },
-                },
-              },
-            },
+        const createView = async (
+          tableId: string,
+          schema: Record<string, ViewFieldMetadata>
+        ) =>
+          await config.api.viewV2.create({
+            name: generator.guid(),
+            tableId,
+            schema,
           })
 
+        const renameColumn = async (table: Table, renaming: RenameColumn) => {
+          const newSchema = { ...table.schema }
+          ;(newSchema[renaming.updated] = {
+            ...table.schema[renaming.old],
+            name: renaming.updated,
+          }),
+            delete newSchema[renaming.old]
```

Comment:
This looks kinda weird, how come this leading `;` is needed?

Reply:
It is not, just some weird autoformatting due some wrong parenthesis 🤷🏽‍♀️